chains/fhenix/addresses.yaml

```diff
@@ -0,0 +1,11 @@
+staticMerkleRootMultisigIsmFactory: "0x3fb9DC84b3c3eF6Ab0BeDC0835027cD2fF9800CE"
```

Comment:
These will need to be ordered alphabetically. If you run yarn lint:fix it will take care of it for you